perllib/FixMyStreet/DB/Result/Contact.pm

```diff
@@ -11,29 +11,29 @@ use base 'DBIx::Class::Core';
 __PACKAGE__->load_components("FilterColumn", "InflateColumn::DateTime", "EncodedColumn");
 __PACKAGE__->table("contacts");
 __PACKAGE__->add_columns(
-  "id",
-  {
-    data_type         => "integer",
-    is_auto_increment => 1,
-    is_nullable       => 0,
-    sequence          => "contacts_id_seq",
-  },
   "body_id",
   { data_type => "integer", is_foreign_key => 1, is_nullable => 0 },
-  "category",
-  { data_type => "text", default_value => "Other", is_nullable => 0 },
   "email",
   { data_type => "text", is_nullable => 0 },
-  "confirmed",
-  { data_type => "boolean", is_nullable => 0 },
-  "deleted",
-  { data_type => "boolean", is_nullable => 0 },
   "editor",
   { data_type => "text", is_nullable => 0 },
   "whenedited",
   { data_type => "timestamp", is_nullable => 0 },
   "note",
   { data_type => "text", is_nullable => 0 },
+  "confirmed",
+  { data_type => "boolean", is_nullable => 0 },
+  "category",
+  { data_type => "text", default_value => "Other", is_nullable => 0 },
+  "deleted",
+  { data_type => "boolean", is_nullable => 0 },
+  "id",
+  {
+    data_type         => "integer",
+    is_auto_increment => 1,
+    is_nullable       => 0,
+    sequence          => "contacts_id_seq",
+  },
```

Comment:
If nothing else has changed, can just silence the noise, it won't make any difference.